cockatrice/src/remoteclient.cpp

```diff
@@ -365,7 +367,9 @@ void RemoteClient::disconnectFromServer()
 QString RemoteClient::getSrvClientID(const QString _hostname)
 {
     QString srvClientID = settingsCache->getClientID();
-    srvClientID += _hostname;
+    QHostInfo hostInfo = QHostInfo::fromName(_hostname);
+    QHostAddress hostAddress = hostInfo.addresses().first();
```

Comment:
well, we can check `QHostInfo::error()` and `QHostInfo::errorString()`. If there's an error on this step we probably can't connect to the server. I think it shouldn't attempt to connect if this fails.